packages/sdk/src/swap/v2/types.ts

```diff
@@ -112,6 +114,7 @@ interface EgressFields {
   failure: Failure | undefined;
   failedAt: number | undefined;
   failedBlockIndex: string | undefined;
+  transactionPayload: z.infer<typeof cfChainsEvmTransaction> | undefined;
```

Comment:
i think this is not completely right, because we are calling `toString()` for some of the properties. but also its a bit hard for an integrator to understand what the type actually is if we define it this way. lets define it explicitly to make that easier:
```suggestion
  transactionPayload: {
    contract: string,
    ...
  }
```